src/controller/api/newsAPIController.js

```diff
@@ -2,27 +2,37 @@ const axios = require('axios');
 const sources = require('../../data/sources.json');
 const sc = require('short-crypt');
 const getDataFromCheerio = require('../../utils/getDataFromCheerio');
-const defaultImage = 'https://raw.githubusercontent.com/MizouziE/energy-prices-api/master/public/img/energy-prices-api-socials.png'
+const defaultImage =
+  'https://raw.githubusercontent.com/MizouziE/energy-prices-api/master/public/img/energy-prices-api-socials.png';
 let articles = {};
 
 sources.forEach(async (source) => {
   try {
     const response = await axios.get(source.site);
     const returnedArticles = getDataFromCheerio(response.data);
 
-    returnedArticles.forEach(async (returned) => { // now we have an array of multiple article's data to iterate through
+    returnedArticles.forEach(async (returned) => {
+      // now we have an array of multiple article's data to iterate through
+      if (source.name === 'CNN'&&!source.base.length) {
+        source.base='https://www.cnn.com';
```

Comment:
This base doesn't need to be set here. There is a key for it in the sources array.